src/wakatime.rs

```diff
@@ -52,17 +36,21 @@ impl WakatimeExtension {
 
         let (platform, arch) = zed::current_platform();
         let asset_name = format!(
-            "wakatime-lsp-{arch}-{os}.tar.gz",
+            "wakatime-lsp-{arch}-{os}.{extension}",
             arch = match arch {
                 zed::Architecture::Aarch64 => "aarch64",
-                zed::Architecture::X86 => "x86",
```

Comment:
i mean wakatime-lsp does not release 32bit arch builds anyways, it might be better to panic/error out if this match arm is reached